scale/recipe/models.py

```diff
@@ -1392,11 +1392,6 @@ def validate_recipe_type_v6(self, name, definition_dict):
                 json = convert_recipe_diff_to_v6_json(diff)
                 diff = rest_utils.strip_schema_version(json.get_dict())
 
-            except RecipeType.DoesNotExist as ex:
```

Comment:
I think we still need to catch the exception, just ignore it instead of adding a warning.